plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/rest/TestIcebergRestCatalogConnectorSmokeTest.java

```diff
@@ -0,0 +1,102 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.iceberg.catalog.rest;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import io.airlift.http.server.testing.TestingHttpServer;
+import io.trino.plugin.iceberg.BaseIcebergConnectorSmokeTest;
+import io.trino.plugin.iceberg.IcebergQueryRunner;
+import io.trino.testing.QueryRunner;
+import io.trino.testing.TestingConnectorBehavior;
+import io.trino.tpch.TpchTable;
+import org.apache.iceberg.FileFormat;
+import org.apache.iceberg.catalog.Catalog;
+import org.apache.iceberg.rest.DelegatingRestSessionCatalog;
+import org.assertj.core.util.Files;
+
+import java.io.File;
+import java.util.Optional;
+
+import static io.trino.tpch.TpchTable.LINE_ITEM;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class TestIcebergRestCatalogConnectorSmokeTest
+        extends BaseIcebergConnectorSmokeTest
+{
+    public TestIcebergRestCatalogConnectorSmokeTest()
+    {
+        super(FileFormat.PARQUET);
+    }
+
+    @Override
+    protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
+    {
+        switch (connectorBehavior) {
+            case SUPPORTS_RENAME_SCHEMA:
+            case SUPPORTS_CREATE_VIEW:
+            case SUPPORTS_COMMENT_ON_VIEW:
+            case SUPPORTS_CREATE_MATERIALIZED_VIEW:
+            case SUPPORTS_RENAME_MATERIALIZED_VIEW:
+                return false;
+            default:
+                return super.hasBehavior(connectorBehavior);
+        }
+    }
+
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        File warehouseLocation = Files.newTemporaryFolder();
+        warehouseLocation.deleteOnExit();
+
+        Catalog backend = RestCatalogTestUtils.backendCatalog(warehouseLocation);
+
+        DelegatingRestSessionCatalog delegatingCatalog = DelegatingRestSessionCatalog
+                .builder().delegate(backend).build();
+
+        TestingHttpServer testServer = delegatingCatalog.testServer();
+        testServer.start();
+        closeAfterClass(testServer::stop);
+
+        return IcebergQueryRunner.builder()
+                .setBaseDataDir(Optional.of(warehouseLocation.toPath()))
+                .setIcebergProperties(
+                        ImmutableMap.<String, String>builder()
+                                .put("iceberg.file-format", "PARQUET")
+                                .put("iceberg.catalog.type", "rest")
+                                .put("iceberg.metastore.rest.uri", testServer.getBaseUrl().toString())
+                                .buildOrThrow())
+                .setInitialTables(ImmutableList.<TpchTable<?>>builder()
+                        .addAll(REQUIRED_TPCH_TABLES)
+                        .add(LINE_ITEM)
```

Comment:
is LINE_ITEM needed here?

Reply:
This initialization was borrowed from the JdbcCatalog work, so it may not be necessary.  I'll remove it (same for the other test as well).